iroh/Cargo.toml

```diff
@@ -42,36 +41,26 @@ futures-sink = "0.3"
 futures-util = "0.3"
 governor = "0.7.0"
 hickory-resolver = { version = "=0.25.0-alpha.4" }
-hostname = "0.4"
 http = "1"
 http-body-util = "0.1.0"
 hyper = { version = "1", features = ["server", "client", "http1"] }
 hyper-util = "0.1.1"
 igd-next = { version = "0.15.1", features = ["aio_tokio"] }
 iroh-base = { version = "0.30.0", default-features = false, features = ["key", "relay"], path = "../iroh-base" }
 iroh-relay = { version = "0.30", path = "../iroh-relay", default-features = false }
-libc = "0.2.139"
 netdev = "0.31.0"
 netwatch = { version = "0.2.0" }
-num_enum = "0.7"
-once_cell = "1.18.0"
 pin-project = "1"
 pkarr = { version = "2", default-features = false, features = [
     "async",
     "relay",
 ] }
 portmapper = { version = "0.3.0", default-features = false }
-postcard = { version = "1", default-features = false, features = [
-    "alloc",
-    "use-std",
-    "experimental-derive",
-] }
 quinn = { package = "iroh-quinn", version = "0.12.0" }
 quinn-proto = { package = "iroh-quinn-proto", version = "0.12.0" }
 quinn-udp = { package = "iroh-quinn-udp", version = "0.5.5" }
 rand = "0.8"
 rcgen = "0.13"
-regex = { version = "1.7.1", optional = true }
```

Comment:
The semvers check correctly points out that this is a breaking change because every optional dependency has an implicit feature of its own name.  So you'll have to list this in the breaking changes section and fixup the title.